proto/remote_execution.proto

```diff
@@ -860,6 +861,35 @@ message Digest {
   int64 size_bytes = 2;
 }
 
+message IOStats {
+  // The number of files downloaded in this tree.
+  int64 file_download_count = 1;
+
+  // The total size of downloaded data.
+  int64 file_download_size_bytes = 2;
+
+  // The time taken to download the tree.
+  int64 file_download_duration_usec = 3;
+
+  // The number of files uploaded in this tree.
+  int64 file_upload_count = 4;
+
+  // The total size of uploaded data.
+  int64 file_upload_size_bytes = 5;
+
+  // The time taken to upload the tree.
+  int64 file_upload_duration_usec = 6;
+}
+
+// Compute usage sampled throughout a task's execution.
+message UsageStats {
+  // Maximum amount of memory used throughout the task's execution.
+  int64 peak_memory_bytes = 1;
+
+  // Total number of CPU-nanoseconds consumed by the task.
+  int64 cpu_nanos = 2;
+}
```

Comment:
nit: could you move these to the bottom of the file? It's not super well-delineated currently but some of the protos in this file were added by us so we'd probably want to keep those separate. I think everything from `ExecutionTask` and below was added by us, if you wouldn't mind also adding a delineating comment while you're there :)